cg/cli/workflow/commands.py

```diff
@@ -89,6 +91,29 @@ def link(context: CGConfig, case_id: str, dry_run: bool):
     analysis_api.link_fastq_files(case_id=case_id)
 
 
+@click.command("metrics-deliver")
+@ARGUMENT_CASE_ID
+@OPTION_DRY
+@click.pass_obj
+def metrics_deliver(context: CGConfig, case_id: str, dry_run: bool) -> None:
+    """Create and validate a metrics deliverables file for given case id.
+    If QC metrics are met it sets the status in Trailblazer to complete.
+    If failed, it sets it as failed and adds a comment with information of the failed metrics."""
+
+    analysis_api: RnafusionAnalysisAPI | TaxprofilerAnalysisAPI = context.meta_apis["analysis_api"]
```

Comment:
both RnafusionAnalysisAPI | TaxprofilerAnalysisAPI are children of NfAnalysisAPI, so might be better to use the later

Reply:
Added [here](https://github.com/Clinical-Genomics/cg/pull/2701/commits/044d3baf389c121697dec2fe65e1e72af79b2980)